### PR TITLE
Linux builds with electron-builder

### PR DESCRIPTION
**Pull request by lipis**
Feel free to download and test them.. :)
- [wire-2.11.2662-x86_64.AppImage](https://wire-app.wire.com/linux/wire-2.11.2662-x86_64.AppImage)
- [wire-2.11.2662-amd64.deb](https://wire-app.wire.com/linux/wire-2.11.2662-amd64.deb)
- [wire-2.11.2662-ia32.AppImage](https://wire-app.wire.com/linux/wire-2.11.2662-ia32.AppImage)
- [wire-2.11.2662-ia32.deb](https://wire-app.wire.com/linux/wire-2.11.2662-ia32.deb)

### CODE REVIEW

**Comment by ConorIA** [comment]
Please make sure to include updated this instructions in the README!

File: package.json [hunk 1]
```diff
@@ -10,16 +10,27 @@
     "localhost": "electron electron --debug=5858 --devtools --enable-logging --env=http://localhost:8888",
     "test": "eslint electron && electron-mocha tests",
     "rebuild-native-modules": "electron-rebuild -m ./electron/node_modules -f",
-    "deb64": "electron-installer-debian --src wrap/build/Wire-linux-x64/ --arch amd64 --config linux.json",
-    "deb32": "electron-installer-debian --src wrap/build/Wire-linux-ia32/ --arch x32 --config linux.json"
+    "linux32": "build --ia32",
```
Comment (develar) [inline package.json]: I suggest to add `-l`, otherwise if you run it on macOS, it will be macOS, not Linux :)

**Comment by lipis** [comment]
Will do once everything is OK and final.. for some reason the `--ia32` is not working.. but will continue tomorrow :)

**Comment by develar** [comment]
> for some reason the --ia32 is not working

Hmm... What do you mean? Please specify error message. Please see [To build app in 32 bit from a machine with 64 bit](https://github.com/electron-userland/electron-builder/wiki/Multi-Platform-Build#to-build-app-in-32-bit-from-a-machine-with-64-bit) (not sure that it is relevant without error message).

**Comment by lipis** [comment]
@develar I have to admit that I was a bit lazy to check it deeper :)

**Comment by lipis** [comment]
screen shot 2016-10-17 at 19 09 24

Is that normal?! The 64bit shows the correct icon but not the 32bit version..

**Comment by develar** [comment]
@lipis I check in the evening (CET).

**Comment by lipis** [comment]
Did anyone have the chance to try any of these builds?! We most likely going to make a release soon and fix issues later if anything.. my only question ATM is regarding the icon of the AppImage :)

**Comment by ConorIA** [comment]
I ran the 64-bit AppImage. No proviene, other than the preferences menu. It was neat!

**Comment by ConorIA** [comment]
🎉🎉🎉🎉🎉

**Comment by lipis** [comment]
### https://wire.com/download/

**Pull request merged**